srcpkgs/libselinux/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'libselinux'
+pkgname=libselinux
+version=3.0
+revision=1
+build_style=gnu-makefile
+make_install_args=SBINDIR=/usr/bin
+hostmakedepends="pkgconf python ruby xz swig"
+makedepends="libsepol pcre-devel"
+depends="libsepol pcre"
+short_desc="SELinux library and simple utilities"
+maintainer="Anjandev Momi <anjan@momi.ca>"
+license="GPL-2.0-only"
+homepage="https://www.nsa.gov/what-we-do/research/selinux/"
+distfiles="https://github.com/SELinuxProject/selinux/releases/download/20191204/${pkgname}-${version}.tar.gz"
+checksum=2ea2b30f671dae9d6b1391cbe8fb2ce5d36a3ee4fb1cd3c32f0d933c31b82433
+
+post_install() {
+	rm -rf ${DESTDIR}/lib/*
```

Comment:
Use this instead:
```sh
mv $DESTDIR/lib/* $DESTDIR/usr/lib
unlink /usr/lib/libselinux.so
ln -s libselinux.so.1 $DESTDIR/usr/lib/libselinux.so
```
And your problem will be solved.
Same comment applied to `libsepol`

Reply:
I haven't tested it but maybe passing `SHLIBDIR=/usr/lib` to `make_install_args` will do the trick as well.

Reply:
@not-chicken If it work, it's better solution, I only look into this template :/

Reply:
Seems to work, upstream had it set to `/lib` 
```
$ xbps-query -f libselinux
...
/usr/lib/libselinux.so.1
...

Reply:
Thank you. I implemented @not-chicken's  changes in the latest patch. It works great!